canonical-websites/www.ubuntu.com
# Fix legal numbering

## Done

* added a .p-list_pattern_lists.scss mixen to creation lists with legal numbers (1., 1.1, 1.2, 1.2.1, etc.. )
* added this to the print stylesheet
* fixed up html to use the new style

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - [/legal/bootstack/service-description](http://0.0.0.0:8001/legal/bootstack/service-description.html)
  - [/legal/short-terms/2015-06-24](http://0.0.0.0:8001/legal/short-terms/2015-06-24.html)
  - [/legal/short-terms/2015-09-09](http://0.0.0.0:8001/legal/short-terms/2015-09-09.html)
  - [/legal/short-terms/2016-05-20](http://0.0.0.0:8001/legal/short-terms/2016-05-20.html)
  - [/legal/short-terms/2016-06-24](http://0.0.0.0:8001/legal/short-terms/2016-06-24.html)
  - [/legal/short-terms/index](http://0.0.0.0:8001/legal/short-terms/index.html)
  - [/legal/short-terms/ja](http://0.0.0.0:8001/legal/short-terms/ja.html)
  - [/legal/ua-miracle-tc-ja](http://0.0.0.0:8001/legal/ua-miracle-tc-ja.html)
  - [/legal/ua-miracle-tc](http://0.0.0.0:8001/legal/ua-miracle-tc.html)
  - [/legal/ubuntu-advantage/service-description-ja/2017-02-09](http://0.0.0.0:8001/legal/ubuntu-advantage/service-description-ja/2017-02-09.html)
  - [/legal/ubuntu-advantage/ua-terms](http://0.0.0.0:8001/legal/ubuntu-advantage/ua-terms.html)
- check that the numbers are accurate with the original pages
- NOTE: *the h2, h3, h4 numbers are not heading sized*

## Issue / Card

Fixes [launchpad bug 1739670](https://bugs.launchpad.net/canonical-website-content/+bug/1739670)
